app/src/main/java/com/eveningoutpost/dexdrip/utils/CheckBridgeBattery.java

```diff
@@ -63,6 +65,34 @@ public static boolean checkBridgeBattery() {
         return lowbattery;
     }
 
+    public static boolean checkForceWearBridgeBattery() {
+
+        boolean lowbattery = false;
+
+        if (!Home.getPreferencesBooleanDefaultFalse("bridge_battery_alerts")) return false;
+        if (!Home.getPreferencesBooleanDefaultFalse("disable_wearG5_on_lowbattery")) return false;
+
+        try {
+            threshold = Integer.parseInt(Home.getPreferencesStringWithDefault("bridge_battery_alert_level", "30"));
+            if (threshold > 5)//give user 5% leeway to begin charging wear device
+                threshold = threshold - 5;
+        } catch (NumberFormatException e) {
+            UserError.Log.e(TAG, "Got error parsing alert level");
+        }
+
+        final int this_level = Home.getPreferencesInt("bridge_battery", -1);
+        UserError.Log.d(TAG, "checkForceWearBridgeBattery threshold:" + threshold + " this_level:" + this_level);
+        if ((this_level > 0) && (threshold > 0)) {
+            if (this_level < threshold) {
+                lowbattery = true;
+                if (notification_showing) {
+                    cancelNotification(NOTIFICATION_ITEM);
```

Comment:
I don't understand why the notification is cancelled here?

Reply:
I cancel the notification because the watch collector is being switched to the phone.  BUT maybe we should just let the user manually cancel it!  Sorry, I'll remove the cancellation.